backend/substrapp/serializers/ledger/computeplan/serializer.py

```diff
@@ -137,6 +138,7 @@ def get_args(self, data):
             'compositeTraintuples': composite_traintuples,
             'aggregatetuples': aggregatetuples,
             'tag': data.get('tag'),
+            'cleanModels': data.get('cleanModels', False),
```

Comment:
```suggestion
            'cleanModels': data.get('clean_models', False),
```

To be consistent with the other endpoints, substra backend should accept snake case fields.

Reply:
Done